kamon-kafka-clients/src/main/resources/reference.conf

```diff
@@ -4,7 +4,10 @@
 
 kamon {
   kafka {
```

Comment:
For consistency with the rest of the modules this should be under `kamon.instrumentation.kafka`. BTW, same goes with the code itself! It should be under the `kamon.instrumentation.kafka` package.